src/dune_pkg/lock_dir.ml

```diff
@@ -261,40 +261,91 @@ module Pkg = struct
   ;;
 end
 
+module Repositories = struct
+  type t =
+    { complete : bool
```

Comment:
I'm also curious since I couldn't get an (complete) idea from reading the code. I feel like it has to do with if we are using all the repos or not?

Reply:
I removed the complete field and it doesn't even appear to be used anywhere.

Reply:
It is serialized into the `lock.dune` file, you should get a compiler error.

`complete` signalizes whether all the repos could be serialized (local paths are not considered serializable).

Reply:
I was able to remove it from everywhere. The serialization logic isn't actually consuming the value.

Reply:
The only consumers of the field are in lock_dir. I suppose it just hasn't been implemented yet.

Reply:
Hmm, weird, I'm currently on vacation and don't have my laptop with me but I remember implementing it.

`can_be_serialized` is a bad name here since there is a predicate for whether repos can be serialized; this merely describes whether all repos could be serialized.

Reply:
Update: lock_dir.ml line 345 determines the value of complete which is defined by serializable.

Reply:
Yes, the value of complete is set there. But I don't see anywhere else where it is read?

Reply:
Line 249